go/libkb/constants.go

```diff
@@ -723,6 +723,6 @@ const (
 const ProfileProofSuggestions = true
 
 const (
-	ExternalURLsBaseKey          = "external_urls"
-	ExternalURLsStellarExchanges = "stellar_exchanges"
+	ExternalURLsBaseKey         = "external_urls"
+	ExternalURLsStellarPartners = "stellar_exchanges"
```

Comment:
sorry to be a pain, but can you change the value of the const to `stellar_partners` too?

just don't want to get confused later...

thanks!

Reply:
done. https://github.com/keybase/keybase/pull/3891